frontend/src/features/wallet/WalletPlugins/WalletPlugins.tsx

```
@@ -0,0 +1,51 @@
+  Group,
+} from "@mantine/core";
+
+import phoenixDLogo from "../../../features/wallet/phoenixDLogo.svg";
+import lndLogo from "../../../features/wallet/lndLogo.svg";
+import clnLogo from "../../../features/wallet/clnLogo.svg";
+import strikeLogo from "../../../features/wallet/strikeLogo.svg";
+
+import { WalletBox } from "./WalletBox";
+
+// export const WalletPlugins = ({ setShowWallet, showWallet }) => {
+export const WalletPlugins = ({ }) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)